### ISSUE TITLE
Some Known Folders won't move

### ISSUE THREAD

**Issue by ChristSent** [ChristSent/OnedriveRoamingProfile#1]
Saved Searches, Start Menu, and App Data Roaming folders don't automatically move due to naming mismatches.

**Comment by cmoneycode** [comment]
The folders I'm seeing that won't move, are folders that don't actually exist (at least by default) on the local user profile. These folders:

SavedGames, SavedSearches, StartMenu, and RoamingAppData.

Are they necessary to even backup? Maybe we leave these commented out and don't bother with them?

**Comment by cmoneycode** [comment]
I am unable to figure out how the ..\userprofile\AppData folder can be redirected, without using ActiveDirectory GPO's.

I tested changing the name syntax in the script, but two problems with that happen.
1. Access to the folder path ..\AppData\<foldername> is denied when trying to Move-Item.
2. Even if it does move an item, it ends up missing from the local folder dir. In other words, the "redirection" feature is not being properly set for the AppData path.

I have not been able to figure out how to set redirection to the AppData folder path.

**Comment by ChristSent** [comment]
You mention AppData, do you mean AppData Roaming. That's the particular
folder I'm trying to redirect. C:\Users\<username>\AppData\Roaming.
Does this folder exhibit the behavior you've described above?

**Comment by cmoneycode** [comment]
Yes, that's right. In my first point (#1) below, it should have a backslash
after AppData...

..\AppData\<any sub folder>

Including the Roaming folder especially. I could not figure out how to
redirect that folder, and even more, I couldn't even move it. It gives an
access denied message.

Sorry I missed your call. Let's chat tomorrow, if possible.

**Comment by ChristSent** [comment]
It looks like at data roaming is successfully re-directing.

 What is not working is when we try to move these listing files in that
data roaming to the new location.

 I noticed this is not working due to 2 problems:
 One. The file path is incorrect if you look where the move takes place the
dollar leaf variable is equal to roaming app data at that point in time
when they try to do the final move. This must be renamed to /app
data/roaming at that point so I put an if statement right after dollar
leave is assigned to say if $leaf contains roaming or actually I did match
 roaming.  If it matches then I change $leaf to equal/app data/roaming then
the move almost works.

2.  The second problem is that when he goes to move the data I cannot
because their files that are in use in at data roaming at the time the move
takes place .  The same problem would happen if you tried to move any other
folder and there was a file that was open and it even if you had a file
open on your desktop you would not be able to move the desktop folder.

  I have to run to work more information to come. Sorry for any miss
spellings I am voice dictating this.
